entrypoint.sh

```diff
@@ -33,4 +37,4 @@ git add -A
 git commit -m "${INPUT_COMMIT_MESSAGE}"
 git push $WIKIP
 
-echo "Finished!"
\ No newline at end of file
+echo "Finished!"
```

Comment:
Line termination logic is changed through the visual editor. https://dev.to/kevinshu/git-and-normalization-of-line-endings-228j. No actual changes, unnecessary to revert.